redirect.json

```
@@ -99,6 +99,13 @@
       "blueprint_url": "https://github.com/home-assistant/core/blob/master/homeassistant/components/automation/blueprints/motion_light.yaml"
     }
   },
+  {
+    "redirect": "calendar",
+    "name": "Calendar",
+    "badge": "Calendar",
+    "description": "show your calendar dashboard",
```

Should we drop the word "dashboard"?

```suggestion
    "description": "show your calendar",
```